MyWorkout.Web/Controllers/ExercisesController.cs

```diff
@@ -56,7 +54,7 @@ public async Task<IActionResult> Details(int? id)
 
             var exercise = await _unitOfWork.ExerciseRepository
                 .Read(id.Value);
-            _unitOfWork.Load(exercise, e => (IEnumerable<Repeat>)e.Repeats);
+            _unitOfWork.Load(exercise, e => (IEnumerable<Exercise>)e.Repeats);
```

Comment:
Why IEnumerable<Excercise> should be IEnumerable<Repeat>

Reply:
I have changed twice maybe, but the error was still there, so I probably forgot to change it back .. Sorry :-(